cartography/sync.py

```diff
@@ -82,9 +91,10 @@ def run(self, neo4j_driver, config):
                     raise  # TODO this should be configurable
                 logger.info("Finishing sync stage '%s'", stage_name)
         logger.info("Finishing sync with update tag '%d'", config.update_tag)
+        return STATUS_SUCCESS
 
 
-def run_with_config(sync, config):
+def run_with_config(sync: Sync, config: Union[Config, argparse.Namespace]) -> int:
```

Comment:
Again, our duck typing implicitly handled the cartography Config object vs argparse Namespace arg here (which was pretty neat), but explicit is better